src/workflow-design-guide/style-guide.rst

```diff
@@ -161,23 +161,8 @@ script:
 Graph String Lines
 ^^^^^^^^^^^^^^^^^^
 
-Multiline ``graph`` strings can be entirely free-form:
-
-.. code-block:: cylc
-
-   [scheduling]
-       [[graph]]
-           R1 = """
-       # Main workflow:
-     FAMILY:succeed-all => bar & baz => qux
-
-       # Housekeeping:
-     qux => rose_arch => rose_prune"""
-
-Whitespace is ignored in graph string parsing, however, so internal graph lines
-can be indented as if part of the :cylc:conf:`flow.cylc` syntax, or even out to the triple
-quotes, if you feel it aids readability (but watch line length with large
-indents; see :ref:`Line Length`):
+Whitespace is ignored in graph string parsing so internal graph lines
+should be indented as if part of the :cylc:conf:`flow.cylc` syntax:
```

Comment:
```suggestion
can be indented as if part of the :cylc:conf:`flow.cylc` syntax:
```
(Should suggests it might be an error not to do so?).

Reply:
Imho it's a style guide so it can be opinionated about what users should do